contrib/actions/update-modules.sh

```diff
@@ -1,39 +1,68 @@
-#!/bin/bash
-set -eEu
-set -o pipefail
+#!/usr/bin/env bash
 
-# file that contains the module information
-MODULES_FILE="modules"
+set -eo pipefail
 
-if ((BASH_VERSINFO[0] < 4)); then
-    echo "This script requires Bash 4.0 or above."
-    exit 1
-fi
+# move to basedir, in case the script is not executed via `make update-modules`
+cd "../../" || exit 1
```

Comment:
@grische the original from the Gluon Script doesn't worked for me in the CI. Maybe you got a better command than this for me?